src/DssSpell.sol

```diff
@@ -75,334 +60,214 @@ contract DssSpellAction is DssAction {
-    address internal constant LOCKSTAKE_ENGINE             = 0x2b16C07D5fD5cC701a0a871eae2aad6DA5fc8f12;
-    address internal constant LOCKSTAKE_CLIP               = 0xA85621D35cAf9Cf5C146D2376Ce553D7B78A6239;
-    address internal constant LOCKSTAKE_CLIP_CALC          = 0xf13cF3b39823CcfaE6C2354dA56416C80768474e;
-
-    // ---------- Wallets ----------
-    address internal constant AAVE_V3_TREASURY   = 0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c;
-    address internal constant EARLY_BIRD_REWARDS = 0x14D98650d46BF7679BBD05D4f615A1547C87Bf68;
+    uint256 internal constant FIVE_PT_TWO_PCT_RATE = 1000000001607468111246255079;
+
+    // --- Math ---
+    uint256 internal constant RAD = 10 ** 45;
+
+    address internal immutable USDS                    = DssExecLib.getChangelogAddress("USDS");
+    address internal immutable SUSDS                   = DssExecLib.getChangelogAddress("SUSDS");
+    address internal immutable ILK_REGISTRY            = DssExecLib.getChangelogAddress("ILK_REGISTRY");
+    address internal immutable LITE_PSM                = DssExecLib.getChangelogAddress("MCD_LITE_PSM_USDC_A");
+    address internal immutable MCD_PAUSE_PROXY         = DssExecLib.getChangelogAddress("MCD_PAUSE_PROXY");
+
+
+    // ---------- BASE Token Bridge ----------
+    // Mainnet addresses
+    address internal constant BASE_GOV_RELAY           = 0x1Ee0AE8A993F2f5abDB51EAF4AC2876202b65c3b;
+    address internal constant BASE_ESCROW              = 0x7F311a4D48377030bD810395f4CCfC03bdbe9Ef3;
+    address internal constant BASE_TOKEN_BRIDGE        = 0xA5874756416Fa632257eEA380CAbd2E87cED352A;
+    address internal constant BASE_TOKEN_BRIDGE_IMP    = 0xaeFd31c2e593Dc971f9Cb42cBbD5d4AD7F1970b6;
+    address internal constant MESSANGER                = 0x866E82a600A1414e583f7F13623F1aC5d58b0Afa;
+    // BASE addresses
+    address internal constant BASE_GOV_RELAY_L2        = 0xdD0BCc201C9E47c6F6eE68E4dB05b652Bb6aC255;
+    address internal constant BASE_TOKEN_BRIDGE_L2     = 0xee44cdb68D618d58F75d9fe0818B640BD7B8A7B7;
+    address internal constant BASE_TOKEN_BRIDGE_IMP_L2 = 0x289A37BE5D6CCeF7A8f2b90535B3BB6bD3905f72;
+    address internal constant USDS_L2                  = 0x820C137fa70C8691f0e44Dc420a5e53c168921Dc;
+    address internal constant SUSDS_L2                 = 0x5875eEE11Cf8398102FdAd704C9E96607675467a;
+    address internal constant SPELL_L2                 = 0x6f29C3A29A3F056A71FB0714551C8D3547268D62;
+    address internal constant MESSANGER_L2             = 0x4200000000000000000000000000000000000007;
+
+    // ---------- Allocator System  ----------
+    address internal constant ALLOCATOR_ROLES          = 0x9A865A710399cea85dbD9144b7a09C889e94E803;
+    address internal constant ALLOCATOR_REGISTRY       = 0xCdCFA95343DA7821fdD01dc4d0AeDA958051bB3B;
+    address internal constant PIP_ALLOCATOR_SPARK_A    = 0xc7B91C401C02B73CBdF424dFaaa60950d5040dB7;
```

Comment:
I use this name because we add it to the CHAINLOG under this name
https://github.com/makerdao/spells-mainnet/pull/435/files#diff-ddf429f8f3b203b411a967680a78a43b6e6ca9aaa574ccc758c38b82c257907fR150

Reply:
Yeah ok, this is the same actual address that will be added to the chainlog for each allocator. So will get a different key for each one.
Anyway, I guess it is fine to leave as is.